tests/phpunit/tests/comment/getCommentCount.php

```diff
@@ -97,4 +98,40 @@ public function test_get_comment_count_post_trashed() {
 		$this->assertSame( 1, $count['post-trashed'] );
 		$this->assertSame( 0, $count['total_comments'] );
 	}
+
+	public function test_get_comment_count_validate_cache() {
```

Comment:
Please ensure that all new tests, include a `@ticket` number. All new tests, should also include a `@covers` as well.

Reply:
Done.